Source/RapyutaSimulationPlugins/Public/Drives/DifferentialDriveComponent.h

```diff
@@ -104,13 +105,17 @@ class RAPYUTASIMULATIONPLUGINS_API UDifferentialDriveComponent : public URobotVe
     float MaxForce = 1000.f;
 
 protected:
+    //! [cm]
     UPROPERTY()
     float WheelPerimeter = 6.28f;
 
+    //! [rad]
     UPROPERTY()
     float PoseEncoderX = 0.f;
+    //! [rad]
     UPROPERTY()
     float PoseEncoderY = 0.f;
+    //! [rad]

```

Comment:
degree

Reply:
As in the comment here, it is meant to be used directly for `cos()/sin()` & `FQuat` so should be rad?
https://github.com/rapyuta-robotics/RapyutaSimulationPlugins/blob/devel/Source/RapyutaSimulationPlugins/Private/Drives/DifferentialDriveComponent.cpp#L113
https://github.com/rapyuta-robotics/RapyutaSimulationPlugins/blob/devel/Source/RapyutaSimulationPlugins/Private/Drives/DifferentialDriveComponent.cpp#L124

Reply:
hmm I see. I thought it is better to variable in cm and deg as much as possible but in this case we can keep this rad. it maybe better to rename as PoseEncoderThetaRad to clarify